recipes-devtools/deepstream/deepstream-6.0_6.0.0-1.bb

```diff
@@ -104,7 +97,6 @@ FILES:${PN}-samples-data = "\
 FILES:${PN}-sources = "${DEEPSTREAM_PATH}/sources"
 
 FILES:${PN}-iothub-client = "${DEEPSTREAM_PATH}/lib/libiothub_client.so"
-FILES:${PN}-azure-edge = "${DEEPSTREAM_PATH}/lib/libazure*"
 FILES:${PN}-triton-plugins = "${DEEPSTREAM_PATH}/lib/triton_backends"
```

Comment:
I don't think this directory exists any longer, so maybe we can drop this package.

I'm a bit concerned, though, that `libnvds_infer_server` and the `libnvdsgst_inferserver` gstreamer plugin both depend on Triton, and was thinking that perhaps we should package those separately.

Reply:
Please check the new implementation